netlify/functions/session.js

```diff
@@ -1,7 +1,8 @@
-const OpenTok = require('daily-opentok-node');
-// const OpenTok = require("opentok");
+const useDaily = process.env.USE_DAILY;
+const OpenTok =
+  useDaily === 'TRUE' ? require('daily-opentok-node') : require('opentok');
 
-exports.handler = async function () {
+exports.handler = async function handler() {
```

Comment:
Was this an eslint thing? It seems a little redundant otherwise, and I _think_ the original format came straight out of Netlify's example. If it's just eslint being obscurely opinionated let's just go with this (what was the rule it's complaining about?)

Reply:
@lazeratops yes, eslint was yelling at me with this warning/error...
```
Unexpected unnamed async function.eslint[func-names](https://eslint.org/docs/rules/func-names)
```